qemu/tests/cfg/hotplug_mem_negative.cfg

```diff
@@ -0,0 +1,43 @@
+- hotplug_mem_negative:
+    type = hotplug_mem_negative
+    mem_fixed = 4096
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 32G
+    guest_numa_nodes = "node0"
+    ppc64,ppc64le:
+        threshold = 0.15
+    policy_mem = default
+    target_mems = "plug1"
+    set_addr = no
+    variants:
+        - overcommit_host_mem:
+            size_mem = 1000G
```

Comment:
For this case, you may need check if this size is bigger than host memory, and increase it if not.

Reply:
ok, corrected.